common/src/main/java/com/sequenceiq/cloudbreak/validation/CrnValidator.java

```diff
@@ -12,16 +12,17 @@
 public class CrnValidator extends AbstractCrnValidator<String> {
 
     @Override
-    protected String getErrorMessageIfServiceOrResourceTypeInvalid(String req, Set<Pair> serviceAndResourceTypePairs) {
```

Comment:
any unit test ?

Reply:
It is transitively tested in CrnValidatorTest (I extended that), but can add tests for this method directly as well.